cni/network/invoker.go

```diff
@@ -17,7 +17,7 @@ type IPAMInvoker interface {
 	Add(IPAMAddConfig) (IPAMAddResult, error)
 
 	// Delete calls to the invoker source, and returns error. Returning an error here will fail the CNI Delete call.
-	Delete(address *net.IPNet, nwCfg *cni.NetworkConfig, args *cniSkel.CmdArgs, options map[string]interface{}) error
+	Delete(addresses []*net.IPNet, nwCfg *cni.NetworkConfig, args *cniSkel.CmdArgs, options map[string]interface{}) error
```

Comment:
Shouldn't this be backward compatible ?
Just thinking out loud, we are changing the data type from string to array. Are we sure that this change in function definition is readily acceptable wherever IPAM invoker is used.

Reply:
As per Paul, they did all tests in all scenarios on both Linux and Windows and it works fine.